scripts/globals/job_utils/geomancer.lua

```diff
@@ -40,9 +40,22 @@ end
 -- TODO: After elements are aligned in the codebase, this should become:
 -- xi.job_utils.geomancer.spawnLuopan = function(player, target, spell, tickEffect, tickPower, targetType)
 xi.job_utils.geomancer.spawnLuopan = function(player, target, modelID, tickEffect, tickPower, targetType, spell)
+    if target == nil then
```

Comment:
```lua
  if target then
        xi.pet.spawnPet(player, xi.pet.id.LUOPAN)
    else
        return
    end
```

Reply:
oof

Reply:
changed now, much better order of opertation